carina-core/src/main/java/com/qaprosoft/carina/core/foundation/report/ZafiraConfigurator.java

```diff
@@ -54,7 +54,7 @@
  */
 public class ZafiraConfigurator implements IConfigurator, ITestRailManager, IQTestManager {
     private static final Logger LOGGER = Logger.getLogger(ZafiraConfigurator.class);
-    
+    private String unUsedVar;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "unUsedVar" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://demo.qaprosoft.com/sonarqube/coding_rules#rule_key=squid%3AS1068)